requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
+cog==0.6.1
+ConfigParser==5.3.0
+defusedxml==0.7.1
+GitPython==3.1.31
```

Comment:
This shouldn't be getting used anymore - where is the import?

Reply:
https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/AddonManager/change_branch.py
and 
https://github.com/FreeCAD/FreeCAD/blob/master/src/Mod/AddonManager/package_details.py

also the string "gitpython" appears in translations

Reply:
I'll strip it out of those, go ahead and remove it here.